.github/workflows/release.yml

```diff
@@ -12,10 +12,10 @@ jobs:
       - name: Prepare repository
         run: git fetch --unshallow --tags
 
-      - name: Use Node.js 18.x
+      - name: Use Node.js 20.x
         uses: actions/setup-node@v3
         with:
-          node-version: 18.x
+          node-version: 20.x
```

Comment:
Related to this.
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#nodejs--20